simulation-ros/src/turtlebot2i/turtlebot2i_moveit/launch/planning_context.launch

```diff
@@ -0,0 +1,25 @@
+<launch>
+  <!-- By default we do not overwrite the URDF. Change the following to true to change the default behavior -->
+  <arg name="load_robot_description" default="false"/>
+
+  <!-- The name of the parameter under which the URDF is loaded -->
+  <arg name="robot_description" default="robot_description"/>
+
+  <!-- Load universal robot description format (URDF) -->
+  <!-- TODO understand this file. For now it isn't launching -->
```

Comment:
Are these comments correct (line 8 and 9)?

Reply:
Yes... As long as the arg `load_robot_description` is false robot's URDF is not loaded inside this file.